### ISSUE TITLE
LL-47 [BUG] Card type not change 

### ISSUE THREAD

**Issue by ILIIH** [ILIIH/Lum-Learning#89]
1) Open create new card 
2) Observer type card 
3) Close create new card 
4) Open create new card  (repeat multiple time)
AR: Card type not changed (Only after restart application)
ER: Card type changed depends on AI guess, but when it isn't trained all type of card should be suggested to user
